pkg/util/gardener/util.go

```diff
@@ -7,9 +7,9 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller"
 )
 
-// IsShootCPNamespace determines whether the format of specified name implies that it is a shoot namespace in a seed
+// IsShootNamespace determines whether the format of specified name implies that it is a shoot namespace in a seed
 // cluster
-func IsShootCPNamespace(namespace string) bool {
+func IsShootNamespace(namespace string) bool {
 	return strings.HasPrefix(namespace, "shoot--")
```

Comment:
A review comment from https://github.com/gardener/gardener-custom-metrics/issues/5:

```suggestion
	return strings.HasPrefix(namespace, "shoot-")
```

Reply:
Thanks for catching this!